src/main/java/org/jenkinsci/plugins/GithubAccessTokenProperty.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2017, CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins;
+
+import hudson.model.UserProperty;
+import hudson.util.Secret;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Remembers the access token used to connect to the Github server
+ *
+ * @since TODO
+ */
+public class GithubAccessTokenProperty extends UserProperty {
+    private final Secret accessToken;
+
+    public GithubAccessTokenProperty(String accessToken) {
```

Comment:
Git**H**ub BTW

Reply:
<img width="252" alt="github-files" src="https://user-images.githubusercontent.com/2662497/32943052-3a2842c0-cb8b-11e7-9f39-658896f830c4.PNG">

Actually I follow the notation _mostly_ used in the rest of the project.

Do you want:
1) change on mine only
2) change all other also
3) nothing

Reply:
I caution on changing too many interfaces unrelated to your change.  I think you should keep your naming consistent with the rest of the project.

If the desire is to rename the classes then that should be treated as a completely separate change.  I imagine there will be upgrade hell from many users who script configurations that would be affected by these kinds of class changes.